src/render/styles/knotx.css.less

```diff
@@ -240,6 +240,11 @@ pre {
   }
   article {
     padding-bottom: 35px;
+
+    .img-centered {
```

Comment:
You don't need to use html markup in markdown to render image properly.
When you change the HTML to MD for images, you can remove this style from here, and change it to:
```
.post {
  img {
    margin-left: auto;
    margin-right:auto;
  }
}
```